hazelcast-spring/pom.xml

```diff
@@ -30,7 +30,7 @@
     </parent>
 
     <properties>
-        <org.springframework.version>3.1.0.RELEASE</org.springframework.version>
+        <org.springframework.version>4.0.2.RELEASE</org.springframework.version>
```

Comment:
Spring 3 already crashes with JCache 1.0.0. I do not think we have to give users a hint about that.